packages/ui-extensions/src/surfaces/point-of-sale/components/shared/BaseTextField.ts

```diff
@@ -1,9 +1,34 @@
+/**
+ * @property `title` the title of the `TextField`.
+ * @property `initialValue` populates the `TextField` with an text initial value.
+ * @property `placeholder` sets a placeholder value for when the `TextField` is empty.
+ * @property `isValid` set whether the current value in the `TextField` is valid.
+ * @property `errorMessage` sets an error message to present to the user.
+ * @property `onChangeText` a callback that is executed every time the `TextField` value changes.
+ */
 export interface BaseTextFieldProps {
+  /**
+   * The title of the `TextField`.
+   */
   title?: string;
-  subtitle?: string;
```

Comment:
I think there are two paths to choose from here:
1. Keep this, document it as doing nothing
2. Remove this, document it in a new section in the `versions` file for 2024.08(?)

Reply:
Yep, in that case, I think you can add the new section to the versions file in this PR so that we don't lose track of it.